spec/index.bs

```diff
@@ -1005,7 +1005,9 @@ This is implemented by
 
 ## Actions reference ## {#api-actions}
 
-A [=host system=] MUST implement each of the following actions. A host system MUST offer the actions under https method only.
+A [=host system=] MUST implement each of the following actions. A host system MUST only accept requests using https, not
+http. The host system MUST not respond to requests made using other protocols (not even with a 400 Error or other http
+status codes).
```

Comment:
Very pick English typo fix: replace "or other http status codes" with "or any other http status code"

Reply:
Thank you!